### PR TITLE
LPS-167866 Created alert for when subtask has different users assigned

### PR DESCRIPTION
**Pull request by rafaelfs98**

### CODE REVIEW

**Comment by rafaelfs98** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/util/search.ts [hunk 1]
```diff
@@ -67,26 +67,28 @@ export const searchUtil = {
 };
 
 export class SearchBuilder {
-	#query: string = '';
+	private query: string = '';
+	private lock: boolean = false;
+	private useURIEncode?: boolean = true;
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/util/search.ts:72]: @rafaelfs98 can you order ascending?

Comment (rafaelfs98) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/util/search.ts:72]: sorted out

**Review by welitonpe — APPROVED** [review]
LGTM :fire:

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/styles/components/_floating_box.scss [hunk 2]
```diff
@@ -8,17 +8,23 @@
 	position: fixed;
 	right: 0;
 	width: 650px;
+	z-index: 99;
```
Comment (samuelleolopes) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/styles/components/_floating_box.scss:11]:
```suggestion
	z-index: 98;
```

**Review by samuelleolopes — COMMENTED** [review]
Check z-index for breadcrumb to be above floatbox

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**